src/vivarium/examples/boids/location_swarming.py

```diff
@@ -0,0 +1,176 @@
+from typing import Any, Dict, List
+
+import numpy as np
+import pandas as pd
+
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.population import SimulantData
+
+
+class Location(Component):
```

Comment:
It's confusing to me that there are two different components called `Location`

Reply:
Yeah, so this is a bit weird -- these two files are actually supposed to represent two _versions_ of the same file. `location.py` is the first version, then (if you were following the tutorial) you would update it to `location_swarming.py`.

One alternative would be to make Swarming a separate component -- is that the direction I should go? Or is this fine? I considered trying to make each _force_ a separate component, but they would need to share acceleration columns, which are currently not in the state table.